consoles/sshVirtsh.pm

```diff
@@ -337,6 +337,42 @@ sub _create_disk ($self, $args, $vmware_openqa_datastore, $file, $name, $basedir
     return $file;
 }
 
+# Check image present in host datastore, otherwhise load it
+sub provide_image_vmware_in_ds ($self, $input_file, $vmware_openqa_datastore, %args) {
+    my $nfs_dir = ($args{backingfile}) ? 'hdd' : 'iso';
+    my $vmware_nfs_datastore = $bmwqemu::vars{VMWARE_NFS_DATASTORE} or die 'Need variable VMWARE_NFS_DATASTORE';
+    my $debug = ($bmwqemu::vars{VMWARE_NFS_DATASTORE_DEBUG} // 0) ? "set -x;" : "";
+    my $base_dir = $bmwqemu::vars{VIRSH_OPENQA_BASEDIR} // "/vmfs/volumes";
+    my $basefile = basename($input_file);
+    # expected name of uncompressed image
+    my $baseimage = basename($input_file) =~ s/\.xz$//r;
+    my $dest_image = "$vmware_openqa_datastore/${baseimage}";
+    # for input file without path, standard folder used
+    my $file_origin = ($input_file eq $basefile) ? "$base_dir/$vmware_nfs_datastore/$nfs_dir/$basefile" : $input_file;
+    # check image is present
+    my $cmd = <<~"EOF";
+$debug
+if test -e "$dest_image"; then
+    echo "Waiting while $input_file is loading:";
+    while (ps -x | grep -E "cp .*$baseimage|xz .*$basefile"|grep -v grep);
+    do sleep 5; done;
+    [ \$\? -eq 0 ] || false;
```

Comment:
This should be just

```suggestion
    [ \$\? -eq 0 ]
```

but actually what for? That should be done by the while-expression already